docs/disconnected-ZTP-flow-hub-deployment.md

```diff
@@ -524,9 +522,6 @@ spec:
     name: mgmt-spoke1
     namespace: mgmt-spoke1
   sshAuthorizedKey: "ssh-rsa adasdlkasjdlklaskdjadoipjasdoiasj root@xxxxXXXXxxx"
-  agentLabelSelector:
```

Comment:
(pls consider) `agentLabelSelector` was moved from `spec` to `status` in its CRD as shown [here](https://github.com/openshift/assisted-service/blob/master/config/crd/bases/agent-install.openshift.io_infraenvs.yaml#L112-L114)... even though OCP doesn't complain either when this CR is applied